multiple-instances.md

```diff
@@ -3,17 +3,114 @@ It is possible to run multiple instances of AIO on one server.
 
 There are two ways to achieve this: The normal way is creating multiple VMs, installing AIO in [reverse proxy mode](./reverse-proxy.md) in each of them and having one reverse proxy in front of them that points to each VM (you also need to [use a different `TALK_PORT`](https://github.com/nextcloud/all-in-one#how-to-adjust-the-talk-port) for each of them). The second and more advanced way is creating multiple users on the server and using docker rootless for each of them in order to install multiple instances on the same server. 
 
-Below is described more in detail how the the second way works.
+## Run multiple AIO instances on the same server inside their own virtual machines

```

Comment:
Can we please move this section below the docker rootless section?

Reply:
Certainly. Will change.

Reply:
Changed in a32f96c72af3d651b9d5e05ca2b2e174eb050b2c